crates/red_knot_python_semantic/src/semantic_index/builder.rs

```diff
@@ -83,6 +89,7 @@ pub(super) struct SemanticIndexBuilder<'db> {
 
     /// Flags about the file's global scope
     has_future_annotations: bool,
+    seen_module_docstring_boundary: bool,
```

Comment:
At some point in the future, it would be useful to combine these flags into a `bitflags` or similar to avoid multiple fields.